### DISCUSSION TITLE
Laser GBRL and ORTUR Laser Master pro2

### DISCUSSION THREAD

**Discussion by timmer23** [arkypita/LaserGRBL#2034] [Q&A]
Hello All,
I'm trying to engrave a logo that measures 4"x8" . When I send it to my Ortur I get a message to reset. I also get a message saying my image is out of bounds.( too large).  I believe this is caused by the settings somewhere but I'm unable to find the solution. 
Thanks for your help.
Tim

**Comment by Tribo-2dot0** [comment]
Verifique nas Configurações grbl os valores de $130 e $131, eles definem o tamanho da sua área de rabalho, em mm.
